addon_service/management/commands/merge_migrated_accounts.py

```diff
@@ -0,0 +1,81 @@
+import logging
+
+from django.core.management import BaseCommand
+from django.db import transaction
+from django.db.models import (
+    Count,
+    F,
+)
+
+from addon_service.authorized_account.citation.models import AuthorizedCitationAccount
+from addon_service.authorized_account.computing.models import AuthorizedComputingAccount
+from addon_service.authorized_account.storage.models import AuthorizedStorageAccount
+from addon_service.osf_models.models import (
+    ExternalAccount,
+    UserToExternalAccount,
+)
+
+
+logger = logging.getLogger(__name__)
+
+services = {
+    "dropbox": "storage",
+    "bitbucket": "storage",
+    "box": "storage",
+    "github": "storage",
+    "googledrive": "storage",
+    "figshare": "storage",
+    "onedrive": "storage",
+    "mendeley": "citations",
+}
+
+
+def get_class(integration_type):
+    if integration_type == "storage":
+        return AuthorizedStorageAccount
+    elif integration_type == "citations":
+        return AuthorizedCitationAccount
+    elif integration_type == "computing":
```

Comment:
Probably don't need to worry about computing so far as we have not migrated any Boa accounts